src/test/java/cat/udl/eps/softarch/academicrecruit/steps/JobApplicationStepsDefs.java

```diff
@@ -46,14 +48,20 @@ public void iRegisterANewJobApplicationNameRequirementsAndDescription(String nam
     }
 
     @And("It has been created a new job application with name {string}, requirements {string} and description {string}")
-    public void itHasBeenCreatedANewJobApplicationWithNameRequirementsAndDescription(String arg0, String arg1, String arg2) {
-    }
+    public void itHasBeenCreatedANewJobApplicationWithNameRequirementsAndDescription(String name, String requirements, String description) throws Exception {
+        List<JobApplication> applicantList = jobApplicationRepository.findByNameContaining(name);
```

Comment:
If it is important that the created job application, in addition to the requested name, also has a defined requirements and description text, you can also include them in a new findBy method that combines them: `findByNameAndRequirementsAndDescription`